From 6ea1d4cae5b37e1a0b6678c6bdcd540b3476a0f6 Mon Sep 17 00:00:00 2001
From: Patrick Delaunay <patrick.delaunay@foss.st.com>
Date: Wed, 2 Jun 2021 10:11:19 +0200
Subject: [PATCH 1/1] doc: usage: reorder commands in index.rst

Reorder alphabetically the command in the index of usage
in U-Boot documentation.

Signed-off-by: Patrick Delaunay <patrick.delaunay@foss.st.com>
Reviewed-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 doc/usage/index.rst | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/doc/usage/index.rst b/doc/usage/index.rst
index c1f9b6a53b..41b247bc62 100644
--- a/doc/usage/index.rst
+++ b/doc/usage/index.rst
@@ -34,12 +34,12 @@ Shell commands
    load
    loady
    mbr
-   mmc
    md
+   mmc
    pstore
    qfw
+   reset
    sbi
+   scp03
    size
    true
-   scp03
-   reset
-- 
2.30.2

